module/modmanager/manager.go

```diff
@@ -1087,7 +1087,8 @@ func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
 
 	// Evaluate the Module's FirstRun path. If there is an error we assume
 	// that the first run script does not exist and we debug log and exit quietly.
-	firstRunPath, markSuccess, err := conf.EvaluateFirstRunPath(packages.LocalPackagesDir(mgr.packagesDir))
+	localPackagesDir := packages.LocalPackagesDir(mgr.packagesDir)
+	firstRunPath, markSuccess, err := conf.EvaluateFirstRunPath(localPackagesDir, logger)
```

Comment:
minor change to help with debugging - this whole section is likely to be refactored in a separate change